FCCee/IDEA/compact/IDEA_o1_v03/detailedMuonSystem_o1_v01.xml

```diff
@@ -0,0 +1,321 @@
+      <endcapDetectorParameters numEndcapDetectorLayers="0" endcapDetectorVolumeThickness="microRWELLVolumeThickEndcap" endcapDetectorLayerInnerRadius="EndcapLayersInnerRadius" endcapDetectorLayerOuterRadius="EndcapLayersOuterRadius" endcapDetectorZOffset="EndcapFirstLayerZOffset+microRWELLVolumeThickEndcap/2.0" />
+
+      <!-- mRWELL chamber -->
+      <layer x="G10_FR4Thick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick/2.0" y_offset="0*mm" z_offset="0*mm" material="G10_FR40x5633439ca570" vis="MuonVis" />
+      <layer x="CuThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick/2.0" y_offset="0*mm" z_offset="0*mm" material="G4_Cu"  vis="MuonVis" />
+      <layer x="GasLayerThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick/2.0" y_offset="0*mm" z_offset="0*mm" material="ARCO2CF4" sensitive="true" vis="MuonVis" />
+      <layer x="Cu2Thick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick/2.0" y_offset="0*mm" z_offset="0*mm" material="G4_Cu"  vis="MuonVis" />
+      <layer x="KaptonThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick/2.0" y_offset="0*mm" z_offset="0*mm" material="Kapton" vis="MuonVis" />
+      <layer x="CarnonFiberThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick/2.0" y_offset="0*mm" z_offset="0*mm" material="CarbonFiber" vis="MuonVis" />
+      <layer x="CuThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick/2.0" y_offset="0*mm" z_offset="0*mm" material="G4_Cu"  vis="MuonVis" />
+      <layer x="CarbonFiber2Thick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick+CarbonFiber2Thick/2.0" y_offset="0*mm" z_offset="0*mm" material="CarbonFiber" vis="MuonVis" />
+      <layer x="CuThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick+CarbonFiber2Thick+CuThick/2.0" y_offset="0*mm" z_offset="0*mm" material="G4_Cu"  vis="MuonVis" />
+      <layer x="SiThick/2.0" y="mRWELLYLength" z="mRWELLZLength" x_offset="-mRWELLTotalThickness/2.0+G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick+CarbonFiber2Thick+CuThick+SiThick/2.0" y_offset="0*mm" z_offset="0*mm" material="G4_Si0x56335d3d1480" vis="MuonVis" />
+ 
```

Comment:
I plan actually to do that, a briefer way to describe the muRWELL chamber in the xml. But, I propose to let it be in the next version, since it requires some changes in the builder.